### DISCUSSION TITLE
part-ii/2021-17/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#403] [Question comments]
Part II 2021 Applied Probability: Paper 2, Section II, $28 K$ 

https://questions.tripos.org/part-ii/2021-17/

**Comment by DannyDuberstein** [comment]
Very unfair question I think,

<details><summary>[spoiler title]</summary>

Turns out in (c) "at least one" means you can just try the first one, it works, and stop. 
Of course though the question is deliberately misleading; when you see "at least one" the natural assumption is that you need to somehow add two quantities and get infinity, or something like that. 

</details>

**Comment by DannyDuberstein** [comment]
In my opinion, the statements in exam questions should be written as they would appear in a theorem. Doing anything else creates artificial difficulty

**Comment by ArthurConmy** [comment]
I also found this very weird; I think because

$q_{xy} = (1 + |x-y|)^2$

will be non-explosive, and have same jump chain as the second chain in question, we can then work with this and show it's non-explosive by using b) and Fubini to show that it suffices to show 

$$ \int_{-\pi}^{+\pi} \frac{1}{\lambda (k)} dk = \infty $$ 

which is true because we can evaluate $ \lambda(k) = O(1/k^2) $.
